kuwala/pipelines/population-density/src/main.py

```diff
@@ -13,7 +13,7 @@
     parser.add_argument('--demographic_groups', help='Demographic groups to be downloaded')
     args = parser.parse_args()
 
-    files, output_dir = Downloader.start(args)
+    files, output_dir, update_date = Downloader.start(args)
```

Comment:
Would be better to follow the same variable name you used at `Downloader.py`
```suggestion
    files, output_dir, updated_date = Downloader.start(args)
```
Same at line no 18

Reply:
Nice catch! Thank you, this is will be useful...